client/my-sites/checkout/src/payment-methods/credit-card/index.tsx

```diff
@@ -61,6 +62,7 @@ const CreditCardLabel: React.FC< { hasExistingCardMethods: boolean | undefined }
 function CreditCardLogos() {
 	return (
 		<PaymentMethodLogos className="credit-card__logo payment-logos">
+			{ hasCheckoutVersion( 'cobadge' ) && <CBLogo /> }
```

Comment:
This works, but when we go ahead and launch this feature, I think it would be good to restrict this by location, since the logo won't mean much to many of our users.